pyproject.toml

```diff
@@ -34,6 +34,8 @@ django-cors-headers = "^4.0.0"
 django-filter = "^2.4.0"
 django-guardian = "^3.0.0"
 django-health-check = { git = "https://github.com/revsys/django-health-check", rev="b0500d14c338040984f02ee34ffbe6643b005084" }  # pragma: allowlist secret
+pypdfium2 = { git = "https://github.com/pypdfium2-team/pypdfium2"}
```

Comment:
pypdfium2 is available from pypi. Is there a reason we are installing the git version. Also it looks like the git version you are installing in this pr isn't version controlled